serverless/serverless-components
# Update GitHub components and add example

## What has been implemented?

Closes https://serverlessteam.atlassian.net/browse/SC-112 & https://serverlessteam.atlassian.net/browse/SC-113

## Steps to verify

```
cd to serverless-components/examples/github-webhook-example

Add your config values

components deploy

Verify API gateway creation

Verify github webhook creation
```

## Todos:
